src/wire_protocol/operations.rs

```diff
@@ -71,7 +55,7 @@ pub enum Message {
         /// database name and a dot separator.
         namespace: String,
         /// The documents to be inserted.
-        documents: Vec<bson::Document>,
+        documents: Vec<u8>,
```

Comment:
This could also be a `Vec<Vec<u8>>`, and is even easier than in Query, because the present callers start with a Vec<bson::Document>, so it's just a question of serializing each to its own vec, or all to one vec.